src/log_manager/log_entry.h

```diff
@@ -14,6 +15,15 @@ struct LogEntry final {
 
     std::string data;
 
+    std::string toString() {
+        return "{termId: " + std::to_string(term_id.getTerm()) + "," +
```

Comment:
Can you use stringstream here? like:
std::stringstream ss;
ss << "{termId: " << term_id.getTerm() 


BTW, can you refine the output as the following format:
```
{
    termId: 12,
    logIndex: 1222;
    data: xsdklfunksuhdgksduf,
}
```